astropy/table/tests/test_item_access.py

```diff
@@ -248,8 +248,9 @@ def test_np_integers(self, table_data):
         py 3.3 failure mode
         """
         t = table_data.Table(table_data.COLS)
-        idxs = np.random.randint(len(t), size=2)
-        t[idxs[1]]
+        row = t[np.int64(0)]
+        if not t.masked:
+            assert row == t[0]
```

Comment:
If something goes wrong and instead of a `Row` we get a `Column` then this test would still succeed. Why not just
```suggestion
        assert type(t[np.int64(0)]) is Row
```
Or maybe there is some parametrized test somewhere already that checks what is the output of `Table` indexing based on the type of the index, and if there is then this test could be added as another parameter.

Reply:
I'm going to go with your suggestion as is